core/net/rpl/rpl-conf.h

```diff
@@ -106,6 +106,22 @@
 #endif /* RPL_CONF_MAX_DAG_PER_INSTANCE */
 
 /*
+ * RPL Default route lifetime
+ * The RPL route lifetime is used for the downward routes and for the default
+ * route. In high density network with DIO suppression activated it may happens
```

Comment:
"in high density network" -> "in a high density network"
happens -> happen (no "s")